### PR TITLE
fix #79482 myadultanimes.com

### LINKED ISSUE #79482: myadultanimes.com
### Issue URL (Annoyance)
[NSFW] [https://myadultanimes.com/conversion-the-arthman/?asgtbndr=1](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fmyadultanimes.com%2Fconversion-the-arthman%2F%3Fasgtbndr%3D1)
### Comment
Username: @wernser412
> popup

### [NSFW] Screenshots
<details>
  <summary>Screenshot 1</summary>

  [Screenshot 1](https://reports-img.adguard.com/FTdmd5Q.png)
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by wout-codes**
#79482

### CODE REVIEW

**Comment by Yuki2718** [comment]
Add `||a.teenspirithentai.com^` too, useful to DNS user.

**Review by Yuki2718 — APPROVED** [review]

File: EnglishFilter/sections/adservers.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||a.teenspirithentai.com^$third-party
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/adservers.txt:8]: to adservers_firstparty please `||a.teenspirithentai.com^`

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -1748,7 +1748,7 @@ hdporn.org##a[href^="https://www.hdporn.org/link/"]
 boxrec.com##.midAdvert
 ||upsieutoc.com/images/2020/08/19/bl.jpg
 uii.io,wordcounter.icu,genpas.icu,shurt.pw##body > iframe
-redgifs.com,imgbaron.com,picbaron.com##iframe[src^="//tsyndicate.com/"]
+myadultanimes.com,redgifs.com,imgbaron.com,picbaron.com##iframe[src^="//tsyndicate.com/"]
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/specific.txt:1751]: It seems this rule does nothing

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -493,7 +493,7 @@ glosbe.com##div[id$="Banner"]
 hdmp4mania1.ga##p > a[href][target="_blank"] > img
 graphicux.com##.gux_smartbanner
 vexfile.com##.title-box > a[href^="https://tracker.vexfile.com/"] > img
-/api/spots/*$domain=yomovies.*|speedostream.com|watchomovies.com|gofilms4u.pro|pornleech.io|hentaihand.com|nhentai.com|filmlinks4u.pro
+/api/spots/*$domain=yomovies.*|speedostream.com|watchomovies.com|gofilms4u.pro|pornleech.io|hentaihand.com|nhentai.com|filmlinks4u.pro|myadultanimes.com
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/specific.txt:496]: These requests will be blocked by `||a.teenspirithentai.com^`, It's better to add this domain to rule which blocks third-party scripts

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]
please also add `||zviybblezmc.com^$third-party` to adservers

File: EnglishFilter/sections/specific.txt (hunk 3 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:496]: I get popups without this rule.

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:1751]: I get popups without this rule.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:1751]: There is a blank leftover with AdGuard Apps without this rule.

File: EnglishFilter/sections/adservers.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:8]: I agree with this.

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/specific.txt:1751]: Ok, I see

File: EnglishFilter/sections/specific.txt (hunk 3 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/specific.txt:496]: Which requests does this rule block?

**Comment by Sergey-Lyapin** [comment]
btw e-mail subscription is missed
Screenshot 2021-04-08 at 14 36 24

File: EnglishFilter/sections/specific.txt (hunk 3 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:496]: Don't need to complicate things. Just use this rule. 

image

**Comment by wout-codes** [comment]
@Sergey-Lyapin why not `myadultanimes.com###custom_html-6`

File: EnglishFilter/sections/specific.txt (hunk 3 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/specific.txt:496]: > These requests will be blocked by ||a.teenspirithentai.com^

Screenshot 2021-04-08 at 14 53 42

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:496]: I agree. But isn't it better to add to this rule? If the site changes adservers, this rule will be working perfectly.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:496]: I agree. But isn't it better to add to this rule? If the site changes adservers, this rule will be working perfectly.

**Comment by Sergey-Lyapin** [comment]
> @Sergey-Lyapin why not `myadultanimes.com###custom_html-6`

Ok, let's use this rule

File: EnglishFilter/sections/specific.txt (hunk 3 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/specific.txt:496]: @wout-codes 
some of third-party ad scripts don't have `/api/spots/` in their URLs, so it's better to add this domain to rule which blocks third-party scripts

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:496]: Ok, bootstrap regex.

**Pull request merged**